src/pem_common_plug.c

```diff
@@ -49,8 +51,13 @@ int pem_valid(char *ciphertext, struct fmt_main *self)
 	keeptr = ctcopy;
 
 	ctcopy += TAG_LENGTH;
-	if ((p = strtokm(ctcopy, "$")) == NULL) // type
+	p = strtokm(ctcopy, "$"); // type
+	if (p == NULL)
 		goto err;
+	if (*p != '1' && !warned++) {
+		fprintf(stderr, "prf algorithm <sha256> is not supported currently\n");
```

Comment:
Please prefix the message with `"Warning: " FORMAT_LABEL ": ` (see example in `bestcrypt_ve_fmt_plug.c` - by the way, that one misses the `!warned` logic somehow...)

Anyway, I hope this is temporary.

Reply:
Ack